frontend/src/components/categories/index.tsx

```diff
@@ -59,6 +62,16 @@ export const Categories = () => {
       toast.success(t("message.item_delete_success"));
     },
   });
+  const { mutateAsync: deleteCategories } = useDeleteMany(EntityType.CATEGORY, {
+    onError: (error) => {
+      toast.error(error.message || t("message.internal_server_error"));
+    },
+    onSuccess: () => {
+      deleteDialogCtl.closeDialog();
+      toast.success(t("message.item_delete_success"));
```

Comment:
I think you need to clear the selection onSuccess.

Reply:
Done :heavy_check_mark:  thanks :pray: